tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -98,7 +106,7 @@ def test_failed_vetext_call_throws_http_exception(mocker, event):
 
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn), (sqsInvokedWithAddOn)])
 def test_failed_vetext_call_throws_general_exception(mocker,  event):
```

Comment:
How is this function directly testing that Exception was raised?

Reply:
if exception wasnt raised then retry sqs wouldnt be calle